newEntry.php

```diff
@@ -1 +1,45 @@
 <?php
+session_start();
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>My Vinyl Collection</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="normalize.css" type="text/css" rel="stylesheet">
+    <link href="stylesheet.css" type="text/css" rel="stylesheet">
+</head>
+<body>
+<div class="title_bar">
+    <div class="logo"><img src="logo1.png" alt="Jenny Logo"></div>
```

Comment:
might need to change format so img is on next line - not sure if thats standard practice